### PR TITLE
Update cosmic_settings.ftl for 'es-419' to align with 'en' file

### PR DESCRIPTION
**Pull request by SotoAugusto**
I had to move some lines around to be 1:1 with the 'en' file.

### CODE REVIEW

File: i18n/es-419/cosmic-settings.ftl [hunk 1]
```diff
@@ -65,96 +196,48 @@ style = Estilo
...
+    .gaps = Espacios alrededor de las ventanas ajustadas
+
+### Appearance: Font
+
+font-config = Configuración de Fuente
+interface-font = Fuente del sistema
+monospace-font = Fuente Monoespaciada
+font-family = Familia
+
+font-weight = Grosor
```
Comment (mmstick) [inline i18n/es-419/cosmic-settings.ftl:215]: Not used anymore

File: i18n/es-419/cosmic-settings.ftl [hunk 2]
```diff
@@ -65,96 +196,48 @@ style = Estilo
...
+    .medium = Medio
+    .semibold = Semi Negrita
+    .bold = Negrita
+    .extra-bold = Extra Negrita
+    .black = Negro
+
+font-style = Estilo
+    .normal = Normal
+    .italic = Itálica
+    .oblique = Oblicua
```
Comment (mmstick) [inline i18n/es-419/cosmic-settings.ftl:229]: Same

File: i18n/es-419/cosmic-settings.ftl [hunk 3]
```diff
@@ -65,96 +196,48 @@ style = Estilo
...
+
+font-stretch = Ancho
+    .condensed = Condensado
+    .normal = Normal
+    .expanded = Expandido
```
Comment (mmstick) [inline i18n/es-419/cosmic-settings.ftl:234]: Same

**Review by mmstick — CHANGES_REQUESTED** [review]
Text for font configuration changed

**Comment by SotoAugusto** [comment]
> Text for font configuration changed

Done requested changes

**Review by mmstick — APPROVED** [review]

**Pull request merged**